resources/views/components/badge.blade.php

```diff
@@ -0,0 +1,14 @@
+<span class="{{ $badgeClasses }}">
```

Comment:
Merge the component attributes to allow adding other classes and attributes like `wire:key`

```suggestion
<span {{ $attributes->class($badgeClasses) }}>
```